stream_cell.go

```diff
@@ -0,0 +1,51 @@
+package xlsx
+
+import (
+	"strconv"
+	"time"
+)
+
+// StreamCell holds the data, style and type of cell for streaming
+type StreamCell struct {
+	cellData  string
+	cellStyle StreamStyle
+	cellType  CellType
+}
+
+// NewStreamCell creates a new StreamCell
+func NewStreamCell(cellData string, cellStyle StreamStyle, cellType CellType) StreamCell{
+	return StreamCell{
+		cellData:  cellData,
+		cellStyle: cellStyle,
+		cellType:  cellType,
+	}
+}
+
+// MakeStringStreamCell creates a new cell that holds string data, is of type string and uses general formatting
```

Comment:
We should rename this to `NewStringStreamCell`  to remain consistent with `NewStreamCell`. The same applies to following functions in this file.

Reply:
Done